src/components/ScoreResult/index.js

```diff
@@ -0,0 +1,54 @@
+import { useEffect, useState } from 'react';
+import {
+  StyledScoreResult,
+  ScoreResultGroup,
+  ScoreResultGroupHeader,
+  Score,
+} from './styled';
+import { usePrevious } from '../../helpers';
+
+const ScoreResult = ({
+  score,
+  actualScore,
+  color,
+  reversed,
+  iterationInterval,
+}) => {
+  const [transitioning, setTransitioning] = useState(false);
+  const previousActualScore = usePrevious(actualScore);
+
```

Comment:
Next three comments on this review is something I think would add an indicator on which answer the user got correct after the count is over. Turns the background Green from #333.
If that background is to much we could change the text instead. What do you think?
Add: 
const [answerColor, setAnswerColor] = useState('#333');